tools/src/test/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationInfoSuite.scala

```diff
@@ -719,7 +719,7 @@ class ApplicationInfoSuite extends FunSuite with Logging {
       val dotDirs = ToolTestUtils.listFilesMatching(tempSubDir, { f =>
```

Comment:
it would be good to validate that the path is what we expect.

Like when we read an existing event log it should come back with the right path, like in the test rapids jar test add something like:

`    assert(apps.head.eventLogPath == "$logDir/rapids_join_eventlog.zstd")
`

it would also be nice to make sure the csv is valid, like trying to put a eventlog path that has a comma in it and make sure we properly replace it so csv is read properly.